packages/module/src/providers/blockfrost.provider.ts

```diff
@@ -98,11 +71,12 @@ export class BlockfrostProvider implements IFetcher, ISubmitter {
       },
     });
 
-    try {
-      return await paginateUTxOs();
-    } catch (error) {
-      return [];
```

Comment:
potentially breaking change, there was silent fail with fallback to `[]` that is now removed and will throw an error instead, but maybe that's fine.